packages/cli/src/commands/proxy/connect.ts

```diff
@@ -0,0 +1,127 @@
+import { ux, Args, Flags } from '@oclif/core'
+import { set } from 'lodash'
+import { tunnelServerFlags, urlFlags } from '@preevy/cli-common'
+import { commands, profileStore, withSpinner, SshConnection } from '@preevy/core'
+import { inspect } from 'util'
+import { formatPublicKey } from '@preevy/common'
+import { spawn } from 'child_process'
+import { connectToTunnelServerSsh } from '../../tunnel-server-client'
+import ProfileCommand from '../../profile-command'
+import { filterUrls, printUrls } from '../urls'
+
+// eslint-disable-next-line no-use-before-define
+export default class Connect extends ProfileCommand<typeof Connect> {
+  static description = 'Tunnel existing local compose application'
+
+  static flags = {
+    ...tunnelServerFlags,
+    ...urlFlags,
+    ...ux.table.flags(),
+    'private-env': Flags.boolean({
+      description: 'Mark all services as private',
+      default: false,
+      required: false,
+    }),
+  }
+
+  static strict = false
+  static hidden = true
+
+  static args = {
+    'compose-project': Args.string({
+      description: 'Compose Project name',
+      required: true,
+    }),
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  async run(): Promise<unknown> {
+    const { flags, args, store } = this
+
+    const pStore = profileStore(store)
+
+    const tunnelingKey = await pStore.getTunnelingKey()
+
+    const tunnelOpts = {
+      url: flags['tunnel-url'],
+      tlsServerName: flags['tls-hostname'],
+      insecureSkipVerify: flags['insecure-skip-verify'],
+    }
+
+    const envId = `local-${Math.random().toString(36).substring(7)}` // local+find_ambient_id_based on compose dir (?)
```

Comment:
why the "local" prefix?

Reply:
Since it's a local environment.
I'll improve it once we've read docker-compose-dir label